lib/mcb/commands/providers/ucas_preferences/show.rb

```diff
@@ -8,7 +8,9 @@
 
   provider = Provider.find_by(provider_code: code)
 
-  if provider.ucas_preferences
+  if !provider
```

Comment:
Is there a cleaner way to right this?
If not, else if, else doesn’t read very well IMO

Reply:
Not sure what a better alternative is.

Reply:
`if provider.blank?` maybe? `unless` might be nicer but I tend to not like it when there's `else`/`elsif` involved.

Reply:
:+1: for `if provider.blank?`.